k8s/pkg/networkservice/clientset/versioned/constants.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 VMware, Inc.
```

Comment:
2019

Reply:
The `clientset` is auto-generated code. maybe better to move it somewhere else. Why not create a new package under `k8s/pks/namespace` and have only this file for a start. I'm sure there will be more code in the future if we wan to get to a more advanced namespace support.